cloud-infrastructure/storage/block-storage/asset/block-volume-backup-and-restore.txt

```diff
@@ -0,0 +1,155 @@
+License
+{----------------------------------------------------------------------------------------------
+Copyright (c) 2023 Oracle and/or its affiliates.
+Licensed under the Universal Permissive License (UPL), Version 1.0.
+See [LICENSE](https://github.com/oracle-devrel/technology-engineering/blob/folder-structure/LICENSE) for more details.
+}----------------------------------------------------------------------------------------------
+
+
+showcase manual block volume backup & restore
+
+prepare environment
+{----------------------------------------------------------------------------------------------
+- FYI: If you use https://notepad-plus-plus.org/ and choose shell as the language you are able to fold/unfold the sections https://npp-user-manual.org/docs/views/#folding 
+- set up the "operate" instance
+	- An Oracle enterprise Linux server with Command Line Interface (CLI) 
+	  see Set up your Server with Resilience by default using CLI https://gitlab.com/hmielimo/cloud-resilience-by-default/#set-up-your-server-with-resilience-by-default-using-cli for details
+	- create and attach block volume: e.g. backup-test (50GB)
+	- create a bucket e.g. backup-test
+}----------------------------------------------------------------------------------------------
+
+Some information 
+{----------------------------------------------------------------------------------------------
+Mounting an Object Storage Bucket as File System on Oracle Linux				https://blogs.oracle.com/cloud-infrastructure/post/mounting-an-object-storage-bucket-as-file-system-on-oracle-linux
+OCI Compute - How to Install s3fs-fuse on Oracle Linux 8 (Doc ID 2938554.1)		https://support.oracle.com/epmos/faces/DocumentDisplay?id=2938554.1
+}----------------------------------------------------------------------------------------------
+
+set needed variable
+{----------------------------------------------------------------------------------------------
+DEVICEoPATH1=/dev/oracleoci/oraclevdc
+DEVICEoPATH2=/dev/oracleoci/oraclevdc1
+MYoSOURCE1=/dev/sdb
+MYoSOURCE2=/dev/sdb1
+DATAoDIR=/mnt/MyBlockVolume.backup-test
+OBJECTDATAoDIR=/mnt/MyObjectStorage.backup-test
+BUCKEToNAME=backup-test
+MYoREGION=eu-frankfurt-1
+MYoNAMESPACE=***
```

Comment:
Add placeholders for `***`, like `<your_namespace>`.

Reply:
done